runtime/tests/integration_test.rs

```diff
@@ -364,6 +371,91 @@ fn reward_block_authors() {
 		});
 }
 
+#[test]
+fn initialize_crowdloan_addresses_with_batch_and_pay() {
+	ExtBuilder::default()
+		.with_balances(vec![
+			(AccountId::from(ALICE), 2_000 * GLMR),
+			(AccountId::from(BOB), 1_000 * GLMR),
+		])
+		.with_collators(vec![(AccountId::from(ALICE), 1_000 * GLMR)])
+		.build()
+		.execute_with(|| {
+			// set parachain inherent data
+			set_parachain_inherent_data();
```

Comment:
Does this mean I need to use specific specs for my tests? Or is it the default specs?

Reply:
ok so default specs, to be udpated